docs/tutorials/kafka-migration.rst

```diff
@@ -0,0 +1,390 @@
+
+After starting the replication flow, MirrorMaker 2 starts moving data between the source and target Kafka clusters in asynchronous mode. To measure how the replication is performing you might want to check the replication lag: the delay in synchronization between the source and target Apache Kafka cluster. Once the lag is `0`, the two Apache Kafka clusters are in sync.
+
+Follow the documentation to review :doc:`how to create services integrations </docs/platform/howto/create-service-integration>`. 
+
+The metric you want to track is called ``kafka_mirrormaker_summary.replication_lag``. The image below showcases a Grafana® dashboard showing the mean MirrorMaker 2 replication lag trending to ``0``.
+
+
+.. image:: /images/tutorials/kafka-migration/mirrormaker-replication-lag.png
+    :alt: Grafana dashboard showing the ``kafka_mirrormaker_summary.replication_lag`` metric
+
+Migrate topic schemas
+---------------------
+
+Apache Kafka topics schemas define the structure of the data in certain topics. They can be migrated two different ways:
+
+#. By replicating the schemas topic stored in Apache Kafka (usually located in the ``_schemas`` topic).
+#. By extracting the schema information from the source and registering in the target environment using the appropriate APIs.
+
+The second option offers much more control over which schemas are migrated. To register the schemas in an Aiven for Apache Kafka service you can:
+
+* Navigate in the `Aiven Console <https://console.aiven.io/>`__, service page, **Schemas** tab.
+* Use the `Karapace rest APIs <https://api.aiven.io/doc/#tag/Service:_Kafka/operation/ServiceSchemaRegistrySubjectVersionPost>`_.
+* Use :doc:`Aiven command line interface </docs/tools/cli>`.
+
+
+Migrate access control list
+---------------------------
+
+Apache Kafka Access Control Lists (ACLs) define how various users are allowed to interact with the topics and schemas. To migrate ACLs we recommend extracting the ACL definition from the source Apache Kafka cluster and recreate the ACL in the target cluster. 
+
+If the target of the migration is Aiven for Apache Kafka, you can define the ACLs with:
+
+* The `Aiven Console <https://console.aiven.io/>`__, service page, **Access Control Lists (ACL)** tab.
+* The dedicated `Aiven REST API <https://api.aiven.io/doc/#tag/Service:_Kafka/operation/ServiceKafkaAclAdd>`__.
+* The dedicated :doc:`Aiven CLI command </docs/tools/cli/service/acl>`.
+
+
+Change client settings
+-----------------------
+
+After the replication flow is running and the schemas and ACLs are in place, you can start pointing producers and consumers to the target Apache Kafka cluster. 
+
+.. Warning::
+
+  To avoid losing Apache Kafka messages during the asynchronous MirrorMaker 2 migration, we suggest to stopping the producers, checking that both the consumer lag in the source system and the MirrorMaker 2 replication lag is ``0`` and then pointing producers and consumers to the target Apache Kafka cluster. 
```

Comment:
```suggestion
  To avoid losing Apache Kafka messages during the asynchronous MirrorMaker 2 migration, we suggest stopping the producers, checking that both the consumer lag in the source system and the MirrorMaker 2 replication lag is ``0``, and then pointing producers and consumers to the target Apache Kafka cluster. 
```